python/cudf/cudf/core/series.py

```diff
@@ -3399,10 +3399,14 @@ def value_counts(
         dtype: int32
         """
 
-        if normalize != False:
-            raise NotImplementedError("Only normalize == False is currently supported")
-        if dropna != True:
-            raise NotImplementedError("Only dropna ==True is currently supported")
+        if normalize is not False:
+            raise NotImplementedError(
+                "Only normalize == False is currently supported"
+            )
+        if dropna is not True:
+            raise NotImplementedError(
+                "Only dropna ==True is currently supported"
```

Comment:
```suggestion
                "Only dropna == True is currently supported"
```

Reply:
We can just apply this after CI passes and I'll admin merge.

Reply:
I was too fast and impatient :)